womtool/src/test/resources/validate/wdl_draft3/invalid/bad_task/error.txt

```diff
@@ -0,0 +1 @@
+Failed to process task definition 'oopsie' (reason 1 of 1): No generated value handle found for consumed value 'UnlinkedIdentifierHook(docker_image)'
```

Comment:
the v3 error message... 🤕

Reply:
Hmm, I bet I can make this a tiny bit friendlier...